ConsentViewController/Classes/Consents/SPUserData.swift

```diff
@@ -121,14 +133,25 @@ public protocol SPObjcUserData {
         gdpr?.applies ?? false
     }
 
-    /// Returns GDPR consent data if any available.
+    /// Returns CCPA consent data if any available.
     /// - SeeAlso: `SPCCPAConsent`
     public func objcCCPAConsents() -> SPCCPAConsent? {
         ccpa?.consents
     }
 
-    /// Indicates whether GDPR applies based on the VendorList configuration.
+    /// Indicates whether CCPA applies based on the VendorList configuration.
     public func objcCCPAApplies() -> Bool {
         ccpa?.applies ?? false
     }
+
+    /// Returns USNat consent data if any available.
+    /// - SeeAlso: `SPUSNatConsent`
+    public func objcUSNatConsents() -> SPUSNatConsent? {
+        usnat?.consents
+    }
+
+    /// Indicates whether GDPR applies based on the VendorList configuration.
```

Comment:
I believe it meant to be `USNat`, not `GDPR`

Reply:
Thanks @Nevazhnovu good catch.